x-pack/plugins/cloud_security_posture/public/pages/configurations/configurations.tsx

```diff
@@ -20,9 +20,11 @@ export const Configurations = () => {
   const location = useLocation();
   const dataViewQuery = useLatestFindingsDataView();
   const getSetupStatus = useCspSetupStatusApi();
-
-  const hasFindings = getSetupStatus.data?.status === 'indexed';
-  if (!hasFindings) return <NoFindingsStates />;
+  const hasFindings =
+    getSetupStatus.data?.indicesDetails[0].status === 'not-empty' ||
+    getSetupStatus.data?.kspm.status === 'indexed' ||
+    getSetupStatus.data?.cspm.status === 'indexed';
+  if (!hasFindings) return <NoFindingsStates posturetype={'cspm'} />;
```

Comment:
why it's not enough to check for `indexed` status here? what the reason for checking also for `'not-empty'`?
The `not-empty` state helps to determine if the indices are `indexed`, i don't see reason for adding this check.